gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/membership/domain_service/ApiPrimaryOwnerDomainService.java

```diff
@@ -61,6 +76,26 @@ public PrimaryOwnerEntity getApiPrimaryOwner(final String organizationId, String
             .orElseThrow(() -> new ApiPrimaryOwnerNotFoundException(apiId));
     }
 
+    public void createApiPrimaryOwnerMembership(String apiId, PrimaryOwnerEntity primaryOwner, AuditInfo auditInfo) {
+        findPrimaryOwnerRole(auditInfo.organizationId())
```

Comment:
Could it be interesting to retrieve the organization more explicitly? (from the signature for instance)
I'm just wondering if we should rely on audit info to get the context of membership creation.